recipes/tempita/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version="0.5.3dev" %}
+
+package:
+  name: tempita
+  version: {{ version }}
+
+source:
+  fn: Tempita-{{ version }}dev.tar.gz
+  url: https://pypi.io/packages/source/T/Tempita/Tempita-{{ version }}.tar.gz
+  md5: 365c3b4f36435e2178902d5619301140
+
+build:
+  number: 0
+  script: python setup.py install --old-and-unmanageable
```

Comment:
Until we pick something different, could we please stick with the standard?

    python setup.py install --single-version-externally-managed --record=record.txt